sell/sell-api.yaml

```
@@ -2054,17 +2033,17 @@ paths:
       summary: Import entities
       description: Use this endpoint to upload files containing entities into Inperium. For example, you can upload your client data exported from other CRMs and that was previously stored in spreadsheets.
       operationId: importFile
-      requestBody:
-        content:
-          multipart/form-data:
-            schema:
-              type: object
-              properties:
-                mappings:
-                  type: string
-                file:
-                  type: string
-                  format: binary
+      parameters:
+        - name: mappings
+          description: This parameter defines mapping between incoming data and Inperium Sell properties.
+          in: query
+          schema:
+            type: string
+        - name: linkToFile
```

```suggestion
        - name: fileUrl
```
I think we used "url" across the API to specify links. Just to be consistent.

---

Fixed